release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-adm-upgrade-status-tp"]
+==== Introducing `oc adm upgrade status` command (Tech Preview)
+
+Previously, customers could retrieve information about the status of their cluster update from the web console and from the CLI. In addition to its primary focus, the `oc adm upgrade` command provided some limited information about the upgrade status. With this release, there is now a tech preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and compliments the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Comment:
I think the right word here is "complement":
```suggestion
Previously, customers could retrieve information about the status of their cluster update from the web console and from the CLI. In addition to its primary focus, the `oc adm upgrade` command provided some limited information about the upgrade status. With this release, there is now a tech preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and complements the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Reply:
I don't see any change with `compliments`.